prepare-videochat.sh

```diff
@@ -359,7 +359,7 @@ if ! has_kernel_module v4l2loopback; then
 fi
 
 # Probe module if not probed yet
-if lsmod | grep v4l2loopback >/dev/null 2>/dev/null; then
+if lsmod | grep -w v4l2loopback >/dev/null 2>/dev/null; then
```

Comment:
I don't think this would result in the correct output. Since `has_kernel_module` also returns 0 when `find /lib/modules/$(uname -r)/ -name "$MODULE.ko" | egrep '.*'` is satisfied, but not necessary that `lsmod | grep -w "$MODULE"` is also satisfied.